### ISSUE TITLE
Add Arrow below to the target view

### ISSUE THREAD

**Issue by ashishgupta2510** [faruktoptas/FancyShowCaseView#128]
### Expected behavior:there is no arrow below target view 
actual behavior:Please Add arrow and give access to add customised arrow

Hi ,
This is a request you to give functionality to add customised arrow below or top to the target view.

Thanks

**Comment by faruktoptas** [comment]
Hi @ashishgupta2510 
You can access `focusCenterX`, `focusCenterY`, `focusHeight` and `focusWidth` parameters. You can add your arrow view in your custom layout by using these parameters.

**Comment by rentalio** [comment]
i always get 0 fancy.focuscenterx fancy.focuscentery

**Comment by rentalio** [comment]
fancyShowCaseView1 = new FancyShowCaseView.Builder(this)
                .focusOn(booking_general_rl_car_transmission_container)
                .enterAnimation(enterAnimation)
                .customView(R.layout.layout_showcaseview_one, new OnViewInflateListener() {
                    @Override
                    public void onViewInflated(View view) {
                        view.findViewById(R.id.btn_guide_one).setOnClickListener(mClickListener);
                        showcaseViewContent(view, fancyShowCaseView1);
                        view.findViewById(R.id.showcase_information).setX(fancyShowCaseView1.getFocusCenterX());
                        view.findViewById(R.id.showcase_information).setY(fancyShowCaseView1.getFocusCenterY());
                        Log.d(TAG, "POSISIY: "+fancyShowCaseView1.getFocusHeight()+" - POSISIX: "+fancyShowCaseView1.getFocusCenterX());
                    }
                })
                .closeOnTouch(false)
                .focusShape(FocusShape.ROUNDED_RECTANGLE)
                .disableFocusAnimation()
                .build();


why Always get 0 x or y

**Comment by rentalio** [comment]
` fancyShowCaseView1 = new FancyShowCaseView.Builder(this)
                .focusOn(booking_general_rl_car_transmission_container)
                .enterAnimation(enterAnimation)
                .customView(R.layout.layout_showcaseview_one, new OnViewInflateListener() {
                    @Override
                    public void onViewInflated(View view) {
                        view.findViewById(R.id.btn_guide_one).setOnClickListener(mClickListener);
                        showcaseViewContent(view, fancyShowCaseView1);
                        view.findViewById(R.id.showcase_information).setX(fancyShowCaseView1.getFocusCenterX());
                        view.findViewById(R.id.showcase_information).setY(fancyShowCaseView1.getFocusCenterY());
                        Log.d(TAG, "POSISIY: "+fancyShowCaseView1.getFocusHeight()+" - POSISIX: "+fancyShowCaseView1.getFocusCenterX());
                    }
                })
                .closeOnTouch(false)
                .focusShape(FocusShape.ROUNDED_RECTANGLE)
                .disableFocusAnimation()
                .build();`

**Comment by faruktoptas** [comment]
@rentalio I will check it. It should return a value after `show()` method is triggered.

**Comment by fajaranugrah** [comment]
this one already on language java? because i'm using java language

**Comment by faruktoptas** [comment]
@fajaranugrah you can use both Java and Kotlin.

**Comment by faruktoptas** [comment]
@fajaranugrah Because you try to show it before fragment is attached. Can you try to move this code under `onViewCreated()` method?

**Comment by faruktoptas** [comment]
@ashishgupta2510 @rentalio I've fixed the problem with the 1.1.5 release. You can access focus position with:

```kotlin
fancy.focusCalculator.circleCenterX
fancy.focusCalculator.circleCenterY

```
Here is a sample usage:
https://github.com/faruktoptas/FancyShowCaseView/blob/ae23f359bbe8bbda47b621ea2b8fcb329081ade4/app/src/main/java/me/toptas/fancyshowcasesample/MainActivity.kt#L228
